frontend/packages/kubevirt-plugin/integration-tests/v2v/vmware/v2v.vmware.import.negative.scenario.incorrect.names.ts

```diff
@@ -0,0 +1,109 @@
+import { asyncForEach, click } from '@console/shared/src/test-utils/utils';
+import { VirtualMachineModel } from '@console/kubevirt-plugin/src/models';
+import { VmwareImportWizard } from '../../tests/models/vmwareImportWizard';
+import { vmwareVMConfig } from './v2v.configs';
+import * as view from '../../views/importWizard.view';
+import { wrongValues } from '../../tests/utils/constants/vm';
+import { clickKebabAction } from '../../views/wizard.view';
+import { NetworkInterfaceDialog } from '../../tests/dialogs/networkInterfaceDialog';
+import { browser } from 'protractor';
+import { DiskDialog } from '../../tests/dialogs/diskDialog';
+
+const {
+  provider,
+  instanceConfig,
+  name,
+  sourceVMName,
+  operatingSystem,
+  flavorConfig,
+  workloadProfile,
+} = vmwareVMConfig;
+
+describe('VMWare Wizard validation, negative tests', () => {
+  const wizard = new VmwareImportWizard();
+
+  beforeEach(async () => {
+    await wizard.openWizard(VirtualMachineModel);
+  });
+
+  afterEach(async () => {
+    await wizard.closeWizard();
+  });
+
+  async function processInstanceConnection() {
```

Comment:
same here

Reply:
I think I've splitt that logic as much as I can, processInstanceConnection differs for rhvImportWizard and vmwareImportWizard, so I will not be able to re-use it somewhere else. I think that is enough here.

Reply:
Done